src/config/experimental.ts

```diff
@@ -57,7 +57,7 @@ export const defaultConfig: Record<string, ExperimentalValue> = {
   [REMOTE_PROMO_SHEETS]: { settings: true, value: false },
   [REMOTE_CARDS]: { settings: true, value: false },
   [POINTS_NOTIFICATIONS_TOGGLE]: { settings: true, value: false },
-  [DAPP_BROWSER]: { settings: true, value: false },
+  [DAPP_BROWSER]: { settings: true, value: true },
```

Comment:
i think we may want to keep this as `false`, bc this could be visible to app store reviewers if the remote config doesn't load properly. maybe do `IS_TEST ? true : false`?